src/MoBi.Assets/AppConstants.cs

```diff
@@ -806,7 +806,7 @@ public static class BarNames
          public static readonly string Views = "Views";
          public static readonly string Add = "Add";
          public static readonly string Edit = "Edit";
-         public static readonly string Project = "Project";
+         public static readonly string ProjectSimulationSettings = "Project Simulation Settings";
```

Comment:
From the screenshot, it feels too long
In which tab is this located in the ribbon?